src/lm_buddy/jobs/_entrypoints/ragas.py

```diff
@@ -0,0 +1,105 @@
+from pathlib import Path
+
+from datasets import load_dataset
+from langchain_community.embeddings import HuggingFaceEmbeddings
+from langchain_openai import ChatOpenAI
+from lmbuddy.integrations.huggingface import HuggingFaceAssetLoader
+from ragas import evaluate
+from ragas.metrics import answer_relevancy, context_precision, context_recall, faithfulness
+
+from lm_buddy.integrations.wandb import artifact_loader
+from lm_buddy.integrations.wandb.artifact_config import WandbArtifactConfig
+from lm_buddy.integrations.wandb.artifact_utils import (
+    ArtifactType,
+    build_directory_artifact,
+    default_artifact_name,
+)
+from lm_buddy.integrations.wandb.run_utils import wandb_init_from_config
+from lm_buddy.jobs.common import EvaluationResult, LMBuddyJobType
+from lm_buddy.jobs.configs import RagasJobConfig
+
+RAGAS_METRICS_MAP = {
```

Comment:
could we get these from `RagasEvaluationConfig`?

Reply:
we don't want to import the jobs directly into the config files, creates issues with pydantic apparently. Implemented it this way as a suggestion from @sfriedowitz

Reply:
The metrics imported from `ragas` are functions. They cannot be serialized. The appropriate metric function must then be looked up by string identifier, which Imtihan has implemented here.